pkg/skaffold/deploy/helm/args.go

```diff
@@ -201,3 +204,54 @@ func (h *Deployer) installArgs(r latest.HelmRelease, builds []build.Artifact, va
 
 	return args, nil
 }
+
+// sanitizeFilePath is used to sanitize filepaths that are provided to the `setFiles` flag
+// helm `setFiles` doesn't work with the unescaped filepath separator (\) for Windows or if there are unescaped tabs and spaces in the directory names.
+// So we escape all odd count occurrences of `\` for Windows, and wrap the entire string in quotes if it has spaces.
```

Comment:
Can you include the information you [mention below](https://github.com/GoogleContainerTools/skaffold/pull/5648/files#r610792583)? 

Does the oddness of backslashes matter?  Can I realistically use three backslashes?

Reply:
I'll do away with this odd-even checks just escaping single `\` characters.